content/en/container_storage_interface.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Container Storage Interface (CSI)
+status: Completed
+category: concept
+---
+
+## What it is
+The Container Storage Interface (CSI) is a standard for exposing arbitrary block and file storage systems to containerized workloads on Container Orchestration Systems (COs) like Kubernetes.
+Container Storage Interface (CSI) is a standardized interface that allows any container technology to "talk" to any cloud native storage system. It functions as an [abstraction](https://glossary.cncf.io/abstraction/) layer between these two technologies, significantly simplifying development work.
```

Comment:
Somehow the informational content is super low. It does not really help to understand what CSI is.